data.js

```diff
@@ -289,6 +289,21 @@ module.exports = [
     twitter: '@wweb_dev',
     tags: ['HTML', 'CSS', 'JavaScript', 'Web Development', 'React', 'Front End', 'Fullstack', 'Next.js', 'Node.js'],
   },
+  {
+    name: "Tim Mouskhelichvili",
+    description:
+      "Hello 👋! I am Tim Mouskhelichvili, a Freelance Developer & Consultant from Montreal 🇨🇦. I write about Front-End Development.",
+    url: "https://timmousk.com/blog/",
+    twitter: "@tim_mousk",
+    tags: ["TypeScript", "JavaScript", "Git", "Web Development", "React"],
+  },
+  {
+    name: 'KnowTechie',
+    description: 'Tech news for people who love tech, but live outside the bubble',
+    url: 'https://knowtechie,com',
```

Comment:
url: 'https://knowtechie.com',

Reply:
Please update the pull request to address this issue.